### PR TITLE
Add serial PKs to column-info-related tables

### PR DESCRIPTION
**Pull request by shen-h**
I followed the naming convention of some other tables that have serial IDs, and named the new serial IDs as `<table_name>_id`.

### CODE REVIEW

File: alembic/versions/0fbe9f4e9934_create_tables.py [hunk 1]
```diff
@@ -190,6 +190,7 @@ def upgrade():
 
     op.create_table(
         'column_info',
+        sa.Column('column_info_id', sa.Integer, primary_key=True),
```
Comment (mollyrossow) [inline alembic/versions/0fbe9f4e9934_create_tables.py]: Do you want `Integer` here or `serial`?

File: alembic/versions/0fbe9f4e9934_create_tables.py [hunk 2]
```diff
@@ -202,6 +203,7 @@ def upgrade():
 
     op.create_table(
         'numeric_column',
+        sa.Column('numeric_column_id', sa.Integer, primary_key=True),
```
Comment (mollyrossow) [inline alembic/versions/0fbe9f4e9934_create_tables.py]: I think this needs to also be a foreign key so you can link it back to column info.

**Comment by mollyrossow** [comment]
After considering this a little more, how about having the field name be `column_info_id` in all five tables to make it clear that it can be used to join across the tables?

**Comment by shen-h** [comment]
@mollyrossow Thanks for the review. Sorry, the last commit was a mess. I just changed them to `column_id`. Will it work?

File: alembic/versions/0fbe9f4e9934_create_tables.py (hunk 1 above)
Comment (shen-h) [reply alembic/versions/0fbe9f4e9934_create_tables.py]: We definitely want serial here, but I looked up the [sqlalchemy documentation](https://docs.sqlalchemy.org/en/latest/dialects/postgresql.html) and it said

> When creating tables, SQLAlchemy will issue the SERIAL datatype for integer-based primary key columns, which generates a sequence and server side default corresponding to the column.

I guess in this way it's created as serial. But I'm checking the information_schema to make sure. Thanks.

**Comment by mollyrossow** [comment]
Yes, I think column_id is a good name.

File: alembic/versions/0fbe9f4e9934_create_tables.py (hunk 1 above)
Comment (shen-h) [reply alembic/versions/0fbe9f4e9934_create_tables.py]: It's confirmed to be serial if that column is defined as INT and PK.

**Comment by shen-h** [comment]
@mollyrossow On second thought, I think `column_id` won't work for the code_frequency table. It will still have a composite key of `column_id` plus `is_null`. Do we want to create another serial PK on the code_frequency table and make it a FK to column_info, in addition to `column_id`? Would that be too complex?

**Comment by mollyrossow** [comment]
You're right, code_frequency needs another serial PK. I think your plan makes sense.

**Comment by shen-h** [comment]
@mollyrossow Since the edits in this branch are a little messy and the dev branch has been updated, I'll close this PR and push another branch for your review.

**Comment by mollyrossow** [comment]
Sounds good.

**Pull request closed without merging**